browser/components/StatsContentCard/StatsContentCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './StatsContentCard.scss';
+import PropTypes from 'prop-types';
+
+
+const StatsContentCard = (props) => {
+  const { godName, imageUrl, featureStats } = props;
+
+  return (
+    <div>
```

Comment:
can we throw a classname on this like stats-content-card

Reply:
Will do and will add more styling